internal/control/control.proto

```
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+// Package fleetedr.control.v1 is the wire contract for the persistent agent
+// control channel: a single bidirectional stream that pushes commands to a
+// connected host in real time and carries command outcomes back. It is the
+// push replacement for the GET /api/commands short-poll (issue #477). The
+// channel authenticates once at connect via the host bearer token in
+// connection metadata; there is no per-frame credential.
+package fleetedr.control.v1;
```

Buf (1.71.0)

[error] Files with package "fleetedr.control.v1" must be within a directory "fleetedr/control/v1" relative to root but were in directory "internal/control".

(PACKAGE_DIRECTORY_MATCH)